src/main/java/hudson/plugins/gradle/injection/NodeLabelItem.java

```diff
@@ -0,0 +1,38 @@
+package hudson.plugins.gradle.injection;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
```

Comment:
Wrong NonNull?

Reply:
This is what is used in the parent class: https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/model/Descriptor.java#L31. But we are not forced to use the same one. Will change to `javax.annotation.Nonnull`